### DISCUSSION TITLE
How safe/viable is it to use refine on top of vite and vitest instead of CRA and jest?

### DISCUSSION THREAD

**Discussion by vinclou** [refinedev/refine#2254] [Q&A] [answered]
P.S I love what I'm seeing in refine docs, but I hate CRA a lot.

**Comment by omeraplak** [comment, marked as answer]
Hey @vinclou ,
I agree with you on the CRA but I think it will take a long time to migrate our 70+ examples to vite :/

Vite example: https://github.com/pankod/refine/tree/next/examples/vite
